dataprofiler/tests/profilers/test_profile_builder.py

```diff
@@ -60,10 +60,21 @@ def setUpClass(cls):
         cls.aws_dataset = pd.read_csv(cls.input_file_path)
         profiler_options = ProfilerOptions()
         profiler_options.set({"data_labeler.is_enabled": False})
+
+        profiler_options_hll = ProfilerOptions()
+        profiler_options_hll.set(
+            {
+                "data_labeler.is_enabled": False,
+                "row_statistics.unique_count.hashing_method": "hll",
+            }
+        )
```

Comment:
adding any tests into `test_row_statistics_options.py` or `test_profiler_options.py` for this new option?

Reply:
although I see in the prior PR you added `hashing_method`

Reply:
new tests in `TestStructuredProfilerRowStatistics` class below